### ISSUE TITLE
QSelect IOS > 10.15.7 - Safari 17.1.2

### ISSUE THREAD

**Issue by sn0rcha** [quasarframework/quasar#16698]
### What happened?

Use clicks on the QSelect field - error is thrown:

```extractFn.value(opt).toLocaleLowerCase is not a function. (In 'extractFn.value(opt).toLocaleLowerCase()', 'extractFn.value(opt).toLocaleLowerCase' is undefined)```

### What did you expect to happen?

No error thrown when clicking on QSelect

### Reproduction URL

https://codepen.io/Ben-Kilah/pen/dyaEKON

### How to reproduce?

Search for a state or click on a the dropdown to select a state

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Safari, iOS

### Quasar info output

```Shell
Operating System - Windows_NT(10.0.22631) - win32/x64
NodeJs - 16.14.2

Global packages
  NPM - 9.6.7
  yarn - Not installed
  @quasar/cli - 2.3.0
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.14.1 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-vite - 1.7.1 -- Quasar Framework App CLI with Vite
  @quasar/extras - 1.16.9 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.47 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.1.6
  pinia - 2.0.30 -- Intuitive, type safe and flexible Store for Vue
  vuex - Not installed
  vite - 2.9.15 -- Native-ESM powered web dev build tool
  eslint - 8.33.0 -- An AST-based pattern checker for JavaScript.
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  @quasar/quasar-app-extension-qcalendar - 4.0.0-beta.16 -- A Quasar App Extension for @quasar/quasar-ui-qcalendar

Networking
  Host - Workhorse
  Ethernet - 192.168.178.126
  VMware Network Adapter VMnet1 - 169.254.195.54
  VMware Network Adapter VMnet8 - 169.254.40.252
  vEthernet (WSL (Hyper-V firewall)) - 172.31.128.1
```

### Relevant log output

```Shell
.value(opt).toLocaleLowerCase is not a function. (In 'extractFn.value(opt).toLocaleLowerCase()', 'extractFn.value(opt).toLocaleLowerCase' is undefined)
```

**Comment by rstoenescu** [comment]
Hi,

I have tested the codepen page & a project with the codepen content on a Safari, albeit on iOS v17.2 (not the reported 10.15.7 to 17.1.2) and there is no issue.

Can you check again after you upgrade to iOS 17.2 please?

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.17.2
